app/scripts/metamask-controller.js

```diff
@@ -1979,11 +1979,9 @@ export default class MetamaskController extends EventEmitter {
           `${this.keyringController.name}:signPersonalMessage`,
           `${this.keyringController.name}:signTypedMessage`,
           `${this.loggingController.name}:add`,
+          `${this.networkController.name}:getNetworkClientById`,
         ],
       }),
-      getAllState: this.getState.bind(this),
```

Comment:
I think we can remove getAllState and getCurrentChainId mocks from `/Users/ariella/git/MetaMask/metamask-extension/app/scripts/controllers/mmi-controller.test.ts` as well

Reply:
Done.